lib/nimbusRecipe.ts

```diff
@@ -240,14 +243,21 @@ export class NimbusRecipe implements NimbusRecipeType {
       branchInfo.nimbusExperiment.startDate,
       branchInfo.nimbusExperiment.proposedDuration,
     );
+    let formattedEndDate;
+    if (branchInfo.nimbusExperiment.endDate) {
+      const endDate = new Date(branchInfo.nimbusExperiment.endDate);
+      endDate.setUTCDate(endDate.getUTCDate() + 1);
+      formattedEndDate = endDate.toISOString().slice(0, 10);
+    }
```

Comment:
If you moved the added clause into a formatDate() function in `experimentUtils.ts`, then this stanza of code would be shorter and more readable/maintainable, because it wouldn't be intermingling different levels of abstraction.